index.html

```diff
@@ -648,25 +649,31 @@ <h3>Additional Properties</h3>
           <h3>Versioning</h3>
           <p>
             Versioning is not provided as an explicit feature of this specification.
-            It is advised to make backwards compatabile changes to schemas, should
-            they be adjusted. Otherwise, it is advised that a new <a>credential schema</a>
-            is created with a unique identifier.
-          </p>
-          <p class="issue" data-number="120">
-            Add language on enabling schema immutability/versioning, such as making use of hashlinks.
+            Implementers are advised to make backwards compatabile changes to schemas, should
+            they be adjusted. Otherwise, it is advised that new <a>credential schemas</a>
+            are created with unique identifiers to avoid processing conflicts.
           </p>
         </section>
+        <section class="informative">
+         <h3>Content Integrity Protection</h3>
+         <p>
+            It is important to make sure that <a>credential schemas</a> have not been tampered with
+            before processing. When making use of the <code>JsonSchemaCredential2023</code> representation
+            of a schema, the credential's associated integrity protection mechanism can act
+            as mutability detection since digital signatures break with content changes.
```

Comment:
I don't quite follow this sentence. What are you trying to say? 

Separately, we already have a normative section about this above; what's the value of this informative section?

Reply:
simply: a digital signature provides information that lets you know if content has been tampered with

the section above is on _processing_ this is intended to be guidance on using schemas

Reply:
I think I get what you're trying to say. Making a suggestion on wording below.